src/main/resources/assets/nochatreports/lang/en_us.json

```diff
@@ -10,5 +10,40 @@
 	"gui.nochatreports.unsafe_server.header": "Caution: Unsafe Server",
 	"gui.nochatreports.unsafe_server.contents": "This server requires all chat messages to be signed, which means they can be reported to Mojang, which may result in a ban from all multiplayer servers. This likely means that the server owner kept \"enforce-secure-profile\" enabled in the server.properties, which is the default since 1.19.1. You can choose to play anyway, but No Chat Reports will not be able to protect you.",
 	"gui.nochatreports.unsafe_server.check": "Do not show again for this server",
-	"gui.nochatrepords.reload_config_tooltip": "Reload No Chat Reports config. Use carefully."
-}
+	"gui.nochatrepords.reload_config_tooltip": "Reload No Chat Reports config. Use carefully.",
+
+	"configuration.NoChatReports.config": "No Chat Reports",
+	"configuration.NoChatReports.category.visual": "Visual",
+	"configuration.NoChatReports.category.technical": "Technical",
+	"configuration.NoChatReports.category.whitelistedServers": "Whitelisted Servers",
+	"option.NoChatReports.demandOnServer": "Demand on server",
```

Comment:
These strings should be ordered the same way as they are displayed in-game, therefore this should come after "always hide report button" etc